src/main/java/edu/kis/powp/jobs2d/features/FeatureManager.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.appbase.Application;
+
+public class FeatureManager {
+
+    private final DrawerFeature drawerFeature;
+    private final CommandsFeature commandsFeature;
+    private final DeviceUsageFeature deviceUsageFeature;
+    private final RecordingFeature recordingFeature;
+    private final DriverFeature driverFeature;
+
+    public FeatureManager(Application app) {
+        this.drawerFeature = new DrawerFeature(app);
+        this.commandsFeature = new CommandsFeature();
+        this.deviceUsageFeature = new DeviceUsageFeature();
+        this.recordingFeature = new RecordingFeature(app, DriverFeature.getDriverManager());
+        this.driverFeature = new DriverFeature(app);
+    }
+
+    public void setupFeatures() {
+        drawerFeature.setup();
+        commandsFeature.setup();
+        deviceUsageFeature.setup();
+        recordingFeature.setup();
+        driverFeature.setup();
+        driverFeature.setUpDriverNameLabelChangeManager();
```

Comment:
FeatureManager should not be aware of methods specific only to a single feature.